types.go

```diff
@@ -9,6 +9,7 @@ import (
 type Result struct {
 	Data                   interface{}                `json:"data"`
 	Errors                 []gqlerrors.FormattedError `json:"errors,omitempty"`
+	Consumer               string                     `json:"consumer,omitempty"`
```

Comment:
This is data that is not related at all to GraphQL but rather to one specific implementation of Graphql. It shouldn't live in this library.